packages/app-desktop/gui/NoteEditor/NoteBody/CodeMirror/CodeMirror.tsx

```diff
@@ -848,6 +848,11 @@ function CodeMirror(props: NoteBodyEditorProps, ref: any) {
 
 	function renderEditor() {
 
+		let matchBracesOptions: boolean | object = Setting.value('editor.autoMatchingBraces');
+		if (matchBracesOptions === true) {
+			matchBracesOptions = { override: true, pairs: '<>()[]{}\'\'""‘’“”（）《》「」『』【】〔〕〖〗〘〙〚〛' };
+		}
```

Comment:
No need for the intermediate assignment, and that way we can make the variable const.

const matchBracesOptions = Setting.value('editor.autoMatchingBraces') ? { override: true, pairs: '<>()[]{}\'\'""‘’“”（）《》「」『』【】〔〕〖〗〘〙〚〛' } : false;

Reply:
I've fixed it in the new commit. Thanks for your review.